src/rpc/rawtransaction.cpp

```diff
@@ -925,7 +925,7 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
     {
         LOCK(cs_main);
         test_accept_res = AcceptToMemoryPool(mempool, state, std::move(tx), &missing_inputs,
-            nullptr /* plTxnReplaced */, false /* bypass_limits */, max_raw_tx_fee, /* test_accept */ true);
+            nullptr /* plTxnReplaced */, false /* bypass_limits */, max_raw_tx_fee, /* test_accept */ true, /* single_threaded_validation */ true);
```

Comment:
Style-Nit: Would be a bit nicer to have consistency in the ordering of argument and comment.

Reply:
The arguments might be wrapped into a struct before they are passed at some point in the future, so I guess this style inconsistency will be solved at that point